app/views/hosts/_form.html.erb

```diff
@@ -102,17 +102,20 @@
       <%= render('hosts/unattended', :f => f) if SETTINGS[:unattended] and @host.managed %>
 
       <div class="tab-pane"  id="params">
-        <h4><%= _('Puppet classes Parameters') %></h4>
-        <p></p>
-        <%= render "puppetclasses/classes_parameters", { :obj => @host } %>
-        <h4><%= _('Included global Parameters via inheritance') %></h4>
-        <p></p>
-        <%= render "common_parameters/inherited_parameters", { :inherited_parameters => @host.host_inherited_params(true) } %>
-        <p></p>
-        <%= render "common_parameters/puppetclasses_parameters", :f => f %>
-        <h4><%= _('Global Host Parameters') %></h4>
-        <p></p>
-        <%= render "common_parameters/parameters", { :f => f, :type => :host_parameters } %>
+        <fieldset>
+          <legend><%= _('Puppet classes Parameters') %></legend>
+          <%= render "puppetclasses/classes_parameters", { :obj => @host } %>
+          <h4><%= _('Puppet class overrides') %></h4>
+          <%= render "common_parameters/puppetclasses_parameters", :f => f %>
+        </fieldset>
+        </br>
```

Comment:
ditto
this is a closing tag, br is a self-closing tag, thus the backslash is in the end